index.js

```diff
@@ -759,3 +762,22 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+  var negatedMethods = {};
+  forEach(source, function (value, key) {
+    var negatedMethod;
+    if (isCallable(value)) {
+      negatedMethod = negatedMethods[key] = function () {
+        return !value.apply(this, arguments);
```

Comment:
to revive the comment - I don't think `this` belongs here - none of the methods in the library use it, and I don't want to enable future uses of it.